### PR TITLE
More XrandR changes

### PR DESCRIPTION
**Pull request by psychon**
This removes some useless font and icon shuffling and uses naughty's destroy callback instead of having its own timer to do this. This has a behaviour change: If you click on the naughty notification, the change is not applied and you stay with the current configuration. Only if you let this actually expire does it apply the change.

### CODE REVIEW

**Comment by psychon** [comment]
I did a force-push and this should no longer have a "big" behavioral change. The new configuration is now still applied when you click the notification, but instead it is now applied immediately instead of waiting for a timeout.

**Comment by blueyed** [comment]
@aajjbb ?

**Comment by aajjbb** [comment]
I do have mixed feelings about having no timeout, I personally like it (it gives me time to cycle again if I'm in a arrangement I don't want). What about making the timeout and argument for `xrandr` ?

**Comment by psychon** [comment]
Let me rephrase what this thing does:

Previously:
- Naughty had a timeout to hide the notification after 4 seconds
- This code had a timeout to apply things after 4 seconds
- When you click the notification, it disappears, but it still takes the full four second until the new state is applied

Now:
- Naughty has a timeout to hide the notification after 4 seconds
- This code doesn't have its own timer, but re-uses the timer that naughty already starts anyway
- When you click the notification, it disappears and the described configuration is applied immediately

That should be the whole difference. There is no "apply the new configuration at the same time that the notification is shown" (which apparently is what you understood).

**Comment by aajjbb** [comment]
Oh, that's great then. I had forgotten about naughty's default timeout. Approved :)

**Comment by blueyed** [comment]
Thanks!
